Makefile

```diff
@@ -19,7 +20,8 @@ publish:
 
 .PHONY: build-content
 build-content:
-	emacs -Q --batch --load=$(BUILD_SRC) --execute "(build/export-all \"$(SOURCE_ORG_FILES)\")" --kill
+	# Temporary change the HOME so that the Emacs installation is independent from the one in the machine.
```

Comment:
The greater (and overkill) alternative would be to use `docker` and make the file being served there 😛  This way I wouldn't had to resort with custom git workflows to have my CI/CD tool. The tool used to build in CI/CD would be literally identical to the one I use to test locally which is ideal.